src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -455,6 +455,22 @@
              Link="Common\System\Net\Http\HttpHandlerDefaults.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'wasi'">
+    <Reference Include="System.Threading.Thread" />
```

Comment:
nit: please move this to the ItemGroup below which has all the `Reference` items